docs: explain attachments in ui mode (#23472)
microsoft/playwright: docs/src/test-ui-mode-js.md

![showing log of tests in ui mode](https://user-images.githubusercontent.com/13063165/234323603-3d9a152d-f9fc-48d8-82d2-26c8c1866abb.png)

### Attachments

> The "Attachments" tab allows you to explore and compare screenshots for when doing [visual regression testing](./test-snapshots.md). It shows the image diff, the actual image and the expected image. When you click on the expected image you can use the slider to slide one image over the other so you can easily see the differences in your screenshots.

Reviewer: ```suggestion
The "Attachments" tab allows you to explore attachments. If you're doing [visual regression testing](./test-snapshots.md), you'll able to compare screenshots by examining image diff, the actual image and the expected image. When you click on the expected image you can use the slider to slide one image over the other so you can easily see the differences in your screenshots.
```
